web.rb

```diff
@@ -208,9 +227,26 @@ def create_customer
       payload = Sinatra::IndifferentHash[JSON.parse(request.body.read)]
   end
 
+  amount = 1099  # Default amount.
```

Comment:
Can we move this out into a `calculate_price(products, shipping)` (or w/e) method?